Makefile

```diff
@@ -2,7 +2,11 @@
 PROJECT_NAME := platform-ref-gcp
 PROJECT_REPO := github.com/upbound/$(PROJECT_NAME)
 
-PLATFORMS ?= linux_amd64 linux_arm64
+# NOTE(turkenh): we need to publish only for linux_amd64 because we are using
+# pkg controllers seems to be looking for that explicitly. Otherwise, it was failing with: 
+# cannot initialize parser backend: failed to fetch package from remote: no child with 
+# platform linux/amd64 in index xpkg.upbound.io/upbound/platform-ref-gcp:v0.2.0-rc.0.4.gdac9a0c
```

Comment:
@turkenh just noting here that this change is not about pushing specifically for `linux_amd64` but rather about _not pushing_ a multi-arch image. We could theoretically use any platform here, but by only using one, we just publish a [manifest](https://explore.ggcr.dev/?image=xpkg.upbound.io%2Fupbound%2Fplatform-ref-gcp%3Av0.2.0), rather than a [manifest list](https://explore.ggcr.dev/?image=xpkg.upbound.io%2Fupbound%2Fplatform-ref-gcp%3Av0.2.0-rc.0.4.gdac9a0c). You'll noticed that in the [config for the single-arch image](https://explore.ggcr.dev/json/xpkg.upbound.io/upbound/platform-ref-gcp@sha256:cf767c6820d229681c9f25e9d3236b0fc35ca62e661183c6f25e1312d2e741e3?mt=application%2Fvnd.docker.container.image.v1%2Bjson&size=555), the arch / OS are not actually set to anything. The place where Crossplane was having trouble was when a manifest list existed (implying a multi-arch image), but not being able to find its own arch in the list. When we only have a manifest published, Crossplane doesn't try to find its arch because their is only one image available.

Reply:
Ah ok, I see. 
Thanks for the explanation @hasheddan.
Opened a PR to fix the comment, #29